.travis.yml

```
@@ -0,0 +1,19 @@
+language: python
+sudo: required
+dist: xenial
+python:
+  - "3.7"
+services:
+# user: postgres, password: <empty>
+  - postgresql
+install:
+  - pip install -r requirements.txt
+  - pip install coverage pylint-django pylint-exit
+before_script:
+  - psql -c 'create database travis_ci;' -U postgres
```

if this is just a testing runner, I don't think the db has to be created either, as the default database exists & the applicaton should just build it's own testing db.
```suggestion
  - psql -c 'create database travis_ci;' -U postgres
```
```suggestion
```